src/py/flwr/client/numpy_client.py

```diff
@@ -50,7 +50,10 @@ def get_parameters(self) -> List[np.ndarray]:
     @abstractmethod
     def fit(
         self, parameters: List[np.ndarray], config: Dict[str, Scalar]
-    ) -> Union[Tuple[List[np.ndarray], int], Tuple[List[np.ndarray], int, Metrics]]:
+    ) -> Union[
+        Tuple[List[np.ndarray], int, Dict[str, Scalar]],
```

Comment:
```suggestion
        Tuple[List[np.ndarray], int, Metrics],
```

Reply:
Discarding this - we've had this in the earlier version of the feature, but I don't want to narrow down the naming too much for now. We can make this more specific at a later stage.